docs/docs/guides/deploy/docker-compose.yaml

```diff
@@ -16,11 +16,28 @@ services:
     ports:
       - '8080:8080'
 
-  db:
+  pg:
     restart: 'always'
     networks:
       - 'zitadel'
-    image: 'cockroachdb/cockroach:v22.1.0'
+    image: postgres:14.4
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready", "-d", "db_prod"]
+      interval: 30s
+      timeout: 60s
+      retries: 5
+      start_period: 80s
+    ports:
+      - '5432:5432'
+
+  crdb:
+    restart: 'always'
+    networks:
+      - 'zitadel'
+    image: 'cockroachdb/cockroach:v22.1.3'
```

Comment:
starting cockroach and postgres to just select one of them to be used by the zitadel config does not make sense to me... and it would not work since the zitadel service has a `depends_on: db`